gapis/resolve/filter.go

```diff
@@ -70,5 +70,26 @@ func buildFilter(
 			return (cmd.CmdFlags().IsExecutedDraw() && len(idx) > 1) || cmd.CmdFlags().IsSubmission()
 		})
 	}
+	if f.GetSuppressHostCommands() {
+		filters = append(filters, func(id api.CmdID, cmd api.Cmd, s *api.GlobalState, idx api.SubCmdIdx) bool {
+			return cmd.CmdFlags().IsSubmission()
```

Comment:
This should not filter-out the `vkQueuePresentKHR` commands.

Reply:
What kind of logic condition can I write to exclude this specific command? Or do I need to give it a new special label so I can identify it for the purposes of exception?

Reply:
@frame_end looks like a suitable way of supporting this.

Reply:
You can cast  cmd to VkQueuePresentKHR and check if it returns ok or not, similar to many operation in transform system.

Reply:
I would prefer the annotation way for now, but If you'd prefer checking for the Vulkan commands explicitly that's OK, too, since we'll remove the API abstraction some time soon. Also include the frame boundary command `vkFrameBoundaryANDROID`.

Reply:
Implemented this with @frame_end which captures the vkFrameBoundaryANDROID case too